docs/en/troubleshooting/acre.md

```diff
@@ -0,0 +1,25 @@
+# ACRE
```

Comment:
I think here we could use some name to distinguish troubleshooting and guide articles
```suggestion
# ACRE - Troubleshooting
```